sql/core/src/main/scala/org/apache/spark/sql/functions.scala

```diff
@@ -8367,8 +8367,17 @@ object functions {
    * @since 3.5.0
    */
   @scala.annotation.varargs
-  def call_function(funcName: String, cols: Column*): Column =
-    withExpr { UnresolvedFunction(funcName, cols.map(_.expr), false) }
+  def call_function(funcName: String, cols: Column*): Column = {
```

Comment:
can we update the API doc? we should highlight that the function name can be qualified using the SQL syntax.

Reply:
Done.